client/src/pages/FloorPlan.js

```diff
@@ -3,25 +3,34 @@ import "./FloorPlan.css";
 
 const Desk = ({ desk, booking }) => {
 	const deskColor = `${booking ? "booked-desk" : "free-desk"}`;
```

Comment:
this is the same as deskColor = booking ? "booked-desk" : "free-desk"; 

also deskClass, not color?